### PR TITLE
Completion of the User Guide documentation and some restructuring work

### PR DESCRIPTION
**Pull request by Lhorus6**
- Add "Enrichment connectors" page
- Add "Workflows and assignation" page
- Complete rewriting of the "Notifications and alerting" page
- Improvement the "Notifier samples" page
- Add "Administration/Notifiers" page
- Complete rewriting of the "Custom dashboards" page
- Add "Widget creation" page
- Improvement of "Customize entities" page
- Minor corrections and improvements elsewhere

### CODE REVIEW

File: docs/usage/dashboards.md [hunk 1]
```diff
@@ -1,21 +1,129 @@
...
 
-`Organizations`, `groups`, or `users` who have access to a dashboard can have 3 levels of access: 
-- `admin` read, write, access management
-- `edit` read and write
-- `view` read-only
 
-When a user creates a custom dashboard, it is only visible to themselves. They then have `admin` access. They can then define who can access it and with what level of rights via the `Manage access` button at the top right of the dashboard page.
+## Dashboard overview
 
+You have the flexibility to tailor the arrangement of [widgets](widgets.md) on your dashboard, optimizing organization and workflow efficiency. Widgets can be intuitively placed to highlight key information. Additionally, dynamically resize widgets from the bottom right corner based on the importance of the information, enabling adaptation to specific analytical needs. This technical flexibility ensures a fluid, visually optimized user experience.
```
Comment (yassine-ouaamou) [inline docs/usage/dashboards.md:8]: Additionally, **you can** or **you have the possibility to** ~~dynamically~~ resize widgets from the bottom right corner based on the importance of the information, enabling adaptation to specific analytical needs.

File: docs/usage/dashboards.md [hunk 2]
```diff
@@ -1,21 +1,129 @@
...
+  "openCTI_version": "5.12.0",
+  "type": "dashboard",
+  "configuration": {
+    "manifest": "eyJ3aWRn(...)bmZpZyI6e319",
+    "name": "Hello Dashboard From Import"
+  }
+}
+```
+
+!!! warning "Source platform-related filters"
```
Comment (yassine-ouaamou) [inline docs/usage/dashboards.md:78]: I don't get what you mean

File: docs/usage/dashboards.md [hunk 3]
```diff
@@ -1,21 +1,129 @@
...
+![Dashboard overview](assets/dashboard-overview.png)
 
-![manage-access-dialog](assets/manage-access-dialog.png)  
-*Manage access window*
 
-It is important to note that a dashboard must have at least one user with `admin` access level.
+## Access control
+
+In OpenCTI, the power to create custom dashboards comes with a flexible access control system, allowing users to tailor visibility and rights according to their collaborative needs.
+Access Restrictions:
```
[the remark this answers is not in the document]
Comment (Lhorus6) [reply docs/usage/dashboards.md:18]: It's a deletion forgetting, good catch

File: docs/usage/dashboards.md (hunk 2 above)
Comment (Lhorus6) [reply docs/usage/dashboards.md:78]: For example, if you have the filter "Label:my-special-label" on the source platform (i.e. where you export the dashboard) and you don't have this label on the receiving platform (i.e. where you import the dashboard)

Do you have a rephrasing suggestion to help people understand?

File: docs/administration/entities.md [hunk 4]
```diff
@@ -2,49 +2,60 @@
...
+This configuration enables the requirement of a reference message on an entity creation or modification. This option is helpful if you want to keep a strong consistency and traceability of your Knowledge and is well suited for manual creation and update.
+
+![Entity configuration](./assets/entity-configuration.png)
 
+
+<a id="workflow-section"></a>
 ## Workflow
 
-For now, OpenCTI have a simple workflow approach.
+For now, OpenCTI have a simple workflow approach. They're represented by the "Processing status" field embedded in each object. By default, this field is disabled for most objects but can be activated through the platform settings:
```
Comment (yassine-ouaamou) [inline docs/administration/entities.md:36]: OpenCTI has*

File: docs/usage/dashboards.md (hunk 2 above)
Comment (yassine-ouaamou) [reply docs/usage/dashboards.md:89]: Ok! I see. I think it's clear the way you said it. I just did not get it

**Pull request merged**